src/sentry/api/endpoints/organization_details.py

```diff
@@ -265,6 +266,8 @@ def save(self):
             org.flags.early_adopter = self.initial_data["isEarlyAdopter"]
         if "require2FA" in self.initial_data:
             org.flags.require_2fa = self.initial_data["require2FA"]
+        if "allowJoinRequests" in self.initial_data:
+            org.flags.disable_join_requests = not self.initial_data["allowJoinRequests"]
```

Comment:
Seems like the flag name is inverted? Why not `org.flags.allow_join_requests`?

Reply:
So it defaults to `False`?

Reply:
Yeah, I can set the BitField default to `65` [here](https://github.com/getsentry/sentry/blob/master/src/sentry/models/organization.py#L141) and the flag would default to `True`, but that would only impact new organizations. The flag would still default to `False` for already existing organizations. @dcramer Is that right -- all new flags should default to `False`?

Reply:
Damn, I would rather not invert the flag name specifically because it defaults to false :(

I would rather run a data migration that sets this flag to true for everyone. Will require a bit of bit math. @wedamija might have some thoughts here.

Reply:
@EvanPurkhiser So just to be clear, we want all orgs to have this new feature enabled by default?

A backfill isn't too hard to do here. Something like 
```
update sentry_organization 
set flags = flags | 64
where id in (select id from sentry_organization where (flags & 64) = 0 limit 1000)
```
Then just query to see if there are rows that need to be backfilled and keep running that statement until we're done. I limit to 1000 just so that we don't do a giant update. fwiw there are a bunch of fields on the org named `disable_*`.

Reply:
Ah okay. I moved this over to an OrganizationOption instead